src/Core/System.php

```diff
@@ -236,6 +237,23 @@ public static function currentLoad()
 		return max($load_arr[0], $load_arr[1]);
 	}
 
+	/**
+	 * Redirects to an external URL (fully qualified URL)
+	 * If you want to route relative to the current Friendica base, use App->internalRedirect()
+	 *
+	 * @param string $url The new Location to redirect
+	 * @throws InternalServerErrorException If the URL is not fully qualified
+	 */
+	public static function externalRedirectTo($url)
```

Comment:
A specific reason why this method has a `-To` suffix but not `App->internalRedirect`?